bitrise.yml

```
@@ -15,3 +15,12 @@ workflows:
   generate_readme:
     steps:
     - git::https://github.com/bitrise-steplib/steps-readme-generator.git@main: { }
+
+  test:
```

General convention is to name this `sample`
```suggestion
  sample:
```